ui/lib/kv/addon/routes/secret/details.js

```diff
@@ -4,23 +4,11 @@
  */
 
 import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
-import { hash } from 'rsvp';
 import { pathIsFromDirectory, breadcrumbsForDirectory } from 'vault/lib/kv-breadcrumbs';
 
 export default class KvSecretDetailsRoute extends Route {
-  @service store;
-  @service secretMountPath;
-
-  model() {
-    // TODO return model for query on kv/data.
-    const backend = this.secretMountPath.get();
-    const { name } = this.paramsFor('secret');
-    return hash({
-      path: name,
-      backend,
-    });
-  }
+  // model passed from parent secret route, if we need to access or intercept
+  // it can retrieved via `this.modelFor('secret')
```

Comment:
This probably broke the breadcrumbs down below. Not a big deal, but just a heads up.

Reply:
Nope :)

Reply:
Resolved model still exists, it's just coming from the parent instead of here